x-pack/plugins/security_solution/server/search_strategy/timeline/factory/events/details/helpers.ts

```diff
@@ -18,39 +19,17 @@ export const getFieldCategory = (field: string): string => {
   return fieldCategory;
 };
 
-export const getDataFromHits = (
-  sources: EventSource,
-  category?: string,
-  path?: string
-): TimelineEventsDetailsItem[] =>
-  Object.keys(sources).reduce<TimelineEventsDetailsItem[]>((accumulator, source) => {
-    const item: EventSource = get(source, sources);
-    if (Array.isArray(item) || isString(item) || isNumber(item)) {
-      const field = path ? `${path}.${source}` : source;
-      const fieldCategory = getFieldCategory(field);
-
-      return [
-        ...accumulator,
-        {
-          category: fieldCategory,
-          field,
-          values: Array.isArray(item)
-            ? item.map((value) => {
-                if (isObject(value)) {
-                  return JSON.stringify(value);
-                }
-
-                return value;
-              })
-            : [item],
-          originalValue: item,
-        } as TimelineEventsDetailsItem,
-      ];
-    } else if (isObject(item)) {
-      return [
-        ...accumulator,
-        ...getDataFromHits(item, category || source, path ? `${path}.${source}` : source),
```

Comment:
I think this part was responsible for a deep parsing of the objects

Before:
![Screenshot](https://user-images.githubusercontent.com/5188868/101292831-2ec28980-3812-11eb-990a-8bbe24a507c0.png)

After:
![Uploading image.png…]()

Reply:
I think that you are right for sure but let me check something first.